mailpoet/lib/Automation/Integrations/MailPoet/Templates/Templates.php

```diff
@@ -0,0 +1,71 @@
+<?php declare(strict_types=1);
+
+namespace MailPoet\Automation\Integrations\MailPoet\Templates;
+
+use MailPoet\Automation\Engine\Workflows\Step;
+use MailPoet\Automation\Engine\Workflows\Workflow;
+use MailPoet\Automation\Integrations\Core\Actions\WaitAction;
+use MailPoet\Automation\Integrations\MailPoet\Actions\SendWelcomeEmailAction;
+use MailPoet\Automation\Integrations\MailPoet\Triggers\SegmentSubscribedTrigger;
+
+class Templates {
+
+  /** @var WaitAction */
+  private $waitAction;
+
+  /** @var SegmentSubscribedTrigger */
+  private $segmentSubscribedTrigger;
+
+  /** @var SendWelcomeEmailAction */
+  private $sendWelcomeEmailAction;
+
+  public function __construct(
+    SegmentSubscribedTrigger $segmentSubscribedTrigger,
+    SendWelcomeEmailAction $sendWelcomeEmailAction,
+    WaitAction $waitAction
+  ) {
+    $this->waitAction = $waitAction;
+    $this->segmentSubscribedTrigger = $segmentSubscribedTrigger;
+    $this->sendWelcomeEmailAction = $sendWelcomeEmailAction;
+  }
+
+  public function delayedEmailAfterSignup(string $name): Workflow {
+    $triggerStep = $this->segmentSubscribedTriggerStep();
+
+    $waitStep = $this->waitStep(60 * 60);
+    $triggerStep->setNextStepId($waitStep->getId());
+
+    $sendEmailStep = $this->sendEmailActionStep();
+    $waitStep->setNextStepId($sendEmailStep->getId());
+    
+    $steps = [
+      $triggerStep,
+      $waitStep,
+      $sendEmailStep,
+    ];
```

Comment:
I added `setNextStepId` because otherwise you kind of have to build workflows in reverse order (if you want to be able to include the nextStepId in the constructor). Maybe there's a better way of handling this too.